### ISSUE TITLE
Does this work for windows?

### ISSUE THREAD

**Issue by ajayyy** [pranav-prakash/QHangups-OSX#3]
I just noticed this, I see your enhancements over the original. Does this work in windows too?

**Comment by pranav-prakash** [comment]
Yea for now it should work (I haven't tested it though). The only mac specific portion I'm planning to add is notifications support (which will use the native osx api). The rest is done using Qt web engine so it should be cross platform.


I'm not too sure how you would go about installing Qt on windows though — especially since you need a fork which has support for qtwebengine (which was deprecated in Qt5).

**Comment by ajayyy** [comment]
Have you added always on top window?

**Comment by pranav-prakash** [comment]
Yup, that is added.

**Comment by ajayyy** [comment]
Ok, that is awesome, I'm going to try it.

**Comment by ajayyy** [comment]
Wait, are you saying I should install Qt4 or Qt5?

**Comment by pranav-prakash** [comment]
You need Qt 5 built with webengine support, along with the pyqt bindings

**Comment by ajayyy** [comment]
Ok, I'm trying to install now, but accidentally downloaded a new version of appdirs. Thanks!

**Comment by ajayyy** [comment]
Yea, since I don't have a version with webengine support, when running qhangupts, nothing happens and the application terminates without error.

**Comment by pranav-prakash** [comment]
Yeah I'm not sure how you go about getting it on Windows. On OSX Homebrew has an option to build from source with Webengine, and on Linux package repos have the appropriate variant.

**Comment by ajayyy** [comment]
Hmm, is __init__.py supposed to be empty?

**Comment by pranav-prakash** [comment]
Yeah it's not used.

**Comment by pranav-prakash** [comment]
If all you're going after is the old hangouts UI, you might also try https://github.com/dhleong/hangr

Which uses electron and should be readily cross platform without mucking around in dependencies.

**Comment by ajayyy** [comment]
Wow, thanks a lot for sharing that. Didn't know that was a thing. Going to try it!
